[lp_intro] update suggestions

longye-tian opened the pull request:
Dear John @jstac,

I have updated some of the lp_intro.md lecture according to #469 (mainly involving typos, and styles)

In particular, the changes include:
## Code
-  round the print in the code 
## Typos and styles
- Unbold non-definition words and use italics for emphases
- Remove capitalization in the titles
- Add period for the first sentence in example 1
- use "`ortools.linear_solver`"
- "... using it's status" -> "... using its status"
- "three year" -> "three-year"
- "Let's us" -> "Let's"
- "LP" -> "Linear programming"
- In the dot point in the "useful transformations" section, remove the capitalization of the second word.
- 'bounds' -> `bounds`
- We did not discuss complementary slackness, so we can delete it.

One thing I find is that the column header is automatically in bold, so I cannot fix it by changing the MD file.
For the suggestions related to materials, I will create a separate issue tracker and pull request.

Best ❤️ 
Longye

longye-tian committed:
[lp_intro] update suggestions

This pull request update some of the the lp_intro.md lecture according to #469 (mainly involving typos, styles)

In particular, the changes include:

- [x] Unbold non-definition words and use italics for emphases
- [x] Remove capitalization in the titles
- [x] Add period for the first sentence in example 1
- [x] use "`ortools.linear_solver`"
- [x] "... using it's status" -> "... using its status"
- Round the print in the format string for printing.
- "three year" -> "three-year"
- "Let's us" -> "Let's"
- "LP" -> "Linear programming"
- In the dot point in the "useful transformations" section, remove the capitalization of the second word.
- 'bounds' -> `bounds`
- We did not discuss complementary slackness, so we can delete it.

One thing I find is that the column header is automatically in bold, so I cannot fix it by changing the md file.

jstac commented:
Many thanks @longye-tian , this is much appreciated.

@mmcky , can you please review?

mmcky commented:
thanks @longye-tian 

> One thing I find is that the column header is automatically in bold

Is this in some code output? Or a part of the document?

longye-tian commented:
> thanks @longye-tian
> 
> > One thing I find is that the column header is automatically in bold
> 
> Is this in some code output? Or a part of the document?

Hi Matt @mmcky ,

It is part of the markdown table, for example in the lp_intro.md, we have the following table,

|          | Product 1| Product 2 |
| :------: | :-------: | :-------: |
| Material |     2     |     5     |
|  Labor   |     4     |     2     |
| Revenue  |     3     |     4     |

The default setting in the md table is that the column header is in bold.  I wish to unbold the column header as according to our style guide, the bold font is for definition. Do you know how to do that? Thank you Matt!

Best,
Longye

mmcky commented:
> > thanks @longye-tian
> > > One thing I find is that the column header is automatically in bold
> > 
> > 
> > Is this in some code output? Or a part of the document?
> 
> Hi Matt @mmcky ,
> 
> It is part of the markdown table, for example in the lp_intro.md, we have the following table,
> 
> Product 1	Product 2
> Material	2	5
> Labor	4	2
> Revenue	3	4
> The default setting in the md table is that the column header is in bold. I wish to unbold the column header as according to our style guide, the bold font is for definition. Do you know how to do that? Thank you Matt!
> 
> Best, Longye

Thanks for explaining this @longye-tian. If this is part of the table rendering then that would be controlled by the theme. 

@jstac would you like all table headers to be not bolded? It is common for them to be bolded but if that interferes with usage from our style guide then we should be able to update the style.

jstac commented:
@mmcky @longye-tian Let's go with the default theme for marking up tables.  It's best to stick with defaults when possible.

Thanks again @longye-tian and thanks @mmcky for the review.  Merging.

longye-tian committed:
[lp_intro] update notations

Dear John @jstac,

This pull request finishes the issue #469 by

- add the notation for $z$, $A_{eq},b_{eq}, A_{ub},b_{ub}$.

- change the orange to black to correspond to the changes in the visualization plot.

Best,
Longye
